va-gov/includes/survey-tools.html

```diff
@@ -1 +1,3 @@
-<script type="text/javascript" src="/js/foresee/{{ buildtype }}.js"></script>
+{% if buildtype == 'vagovstaging' or buildtype == 'preview' %}
+  <script type="text/javascript" src="/js/foresee/{{ buildtype }}.js"></script>
+{% endif %}
```

Comment:
👍This also reminds me that I think we need Foresee script, `vagovprod.js`, to configure Foresee on `www.va.gov`.

CC: @BBarone1

Reply:
thanks @ncksllvn is that something that we can add here or do we need a separate implementation ticket for a new PR?

Reply:
@ncksllvn Do we want to do this with this ticket or create a new ticket. If we are doing this in this ticket who is the one that is going to implement it. I am not familiar with the Foresee service so I would need some documentation if I were to implement it.

Reply:
I think we just need to make a copy of the preview one named vagovprod.js. The current preview one has the production, va.gov changes in it.